src/shared/logs-show.c

```diff
@@ -306,7 +306,7 @@ static int output_timestamp_realtime(FILE *f, sd_journal *j, OutputMode mode, Ou
                 return -EINVAL;
         }
 
-        if (mode == OUTPUT_SHORT_FULL) {
+        if (mode == OUTPUT_SHORT_FULL || mode == OUTPUT_WITH_UNIT) {
```

Comment:
hmm, one more change, can you change this to:

```c
if (IN_SET(mode, OUTPUT_SHORT_FULL, OUTPUT_WITH_UNIT)) { 
```

that's how we usually write this now. It's prettier to look at (and also likely generates nicer code)

sorry for not noticing this earlier